packages/schema/exported-schema.json

```diff
@@ -1063,6 +1063,87 @@
       },
       "additionalProperties": false
     },
+    "BasicHookToPollOperationSchema": {
+      "id": "/BasicHookToPollOperationSchema",
+      "description": "Represents the inbound mechanics of hook to poll style triggers. Defers to list for fields.",
+      "type": "object",
+      "required": ["performList", "performSubscribe", "performUnsubscribe"],
+      "properties": {
+        "type": {
+          "description": "Must be explicitly set to `\"hook_to_poll\"`.",
+          "type": "string",
+          "enum": ["hook_to_poll"],
+          "required": {
+            "type": "replace",
+            "value": "**yes** (with exceptions, see description)"
+          }
+        },
+        "resource": {
+          "description": "Optionally reference and extends a resource. Allows Zapier to automatically tie together samples, lists and hooks, greatly improving the UX. EG: if you had another trigger reusing a resource but filtering the results.",
+          "$ref": "/KeySchema"
+        },
```

Comment:
Should I take this bit out? Not super sure how it works, but could be a bit complicated to support resources?

Reply:
It's doable, but it will take updates in core too. I don't love resources, so I vote circle back to them as needed